src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ReviewCard = ({ review }) => {
+  console.log(review);
+  return (
+    <li className="reviewCard">
```

Comment:
you've got a good grasp of semantic html and are applying it very well throughout